test/controllers/check.your.answers.controller.spec.ts

```diff
@@ -623,10 +632,501 @@ describe("GET tests", () => {
   });
 });
 
+describe("GET with url params tests tests", () => {
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    mockIsActiveFeature.mockReset();
```

Comment:
you can remove the mockIsActiveFeature.mockReset();
and
process.env.FEATURE_FLAG_ENABLE_REDIS_REMOVAL_27092023 = "false";
from the before each, as the tests are the same as the other GET tests and they just use jest.clearAllMocks();